cmd/jaeger/badger_config.yaml

```diff
@@ -20,12 +20,14 @@ extensions:
         ephemeral: false
         maintenance_interval: 5
         metrics_update_interval: 10
+        span_store_ttl: 720h
       badger_archive:
         directory_key: "/tmp/jaeger_archive/"
         directory_value: "/tmp/jaeger_archive/"
         ephemeral: false
         maintenance_interval: 5
         metrics_update_interval: 10
+        span_store_ttl: 720h
```

Comment:
30d, why do this in hours?

Reply:
because i was getting this error :
```
 * error decoding 'badger[badger_main].span_store_ttl': time: unknown unit "d" in duration "30d"
2024/04/21 20:19:40 failed to get config: cannot unmarshal the configuration: 1 error(s) decoding:
```